### ISSUE TITLE
INSPIRE Monitoring / SpatialDataSet / MdAccessibility missing

### ISSUE THREAD

**Issue by fxprunayre** [titellus/daobs#34]
```
		<xs:element name="viewUuid" type="xs:string" minOccurs="0"/>
			<xs:element name="downloadUuid" type="xs:string" minOccurs="0"/>
```

eg. <name>Ecotopen vlakken eerste cyclus</name>
